app/templates/components/forms/reset-password-form.hbs

```diff
@@ -21,6 +21,11 @@
               <div class="ui left icon input">
                 <i class="key icon"></i>
                 {{input type='password' name='password' placeholder=(t 'password') value=password}}
+                  <div class="ui small basic icon buttons">
+                    <button type="button" class="ui button" {{action 'showPassword' 'New'}}>
```

Comment:
This won't work, How is the form supposed to get the `showPassword` Action ?

Reply:
I am committed the action and indentation error . Just pushing it after preview